src/pages/Overview/Privacy/Privacy.jsx

```diff
@@ -0,0 +1,495 @@
+import React, { Component } from 'react'
+import { Route, Switch } from 'react-router-dom'
+
+//import NavigationBar from '../NavigationBar/NavigationBar'
+//import { stickyFooter } from '../Footer/Footer'
+import './Privacy.css'
+
+class PrivacyItem extends Component {
+  render() {
+    return (
+      <div className='PrivacyItem' style={{ float: this.props.float }}>
+        {this.props.text}
+      </div>
+    )
+  }
+}
```

Comment:
seems like this isnt used anywhere

Reply:
Will update in next PR! Updated locally!